index.html

```diff
@@ -832,7 +832,7 @@ <h3>Presentations</h3>
 information <a>graphs</a>. The first <a>graph</a> expresses the
 <a>verifiable presentation</a> itself, which contains
 <span class="color-text" style="color:#b4a7d6;">presentation metadata</span>.
-The <code>verifiablePresentation</code> property in the <a>graph</a> refers to
+The <code>VerifiablePresentation</code> property in the <a>graph</a> refers to
```

Comment:
```suggestion
The <code>verifiableCredential</code> property in the <a>graph</a> refers to
```

@brentzundel is correct.

Reply:
Actually, I got confused because of the name. That is the correct one!